plugins/rest/rest_test.go

```diff
@@ -1277,6 +1277,38 @@ func TestOauth2ClientCredentialsJwtAuthentication(t *testing.T) {
 	}
 }
 
+// https://github.com/open-policy-agent/opa/issues/3255
+func TestS3SigningInstantiationInitializesLogger(t *testing.T) {
+	config := fmt.Sprintf(`{
```

Comment:
[nit] There doesn't seem to be a need for Sprintf here.
```suggestion
	config := `{
```